SimTracker/TrackAssociatorProducers/python/quickTrackAssociatorByHits_cfi.py

```diff
@@ -8,30 +8,26 @@
 	Purity_SimToReco = cms.double(0.75),
 	ThreeHitTracksAreSpecial = cms.bool(True),
         PixelHitWeight = cms.double(1.0),
-	associatePixel = cms.bool(True),
-	associateStrip = cms.bool(True),
-        pixelSimLinkSrc = cms.InputTag("simSiPixelDigis"),
-        stripSimLinkSrc = cms.InputTag("simSiStripDigis"),
         useClusterTPAssociation = cms.bool(True),
         cluster2TPSrc = cms.InputTag("tpClusterProducer")
 )
 
+quickTrackAssociatorByHitsTrackerHitAssociator = quickTrackAssociatorByHits.clone(
+    useClusterTPAssociation = False,
+    associateStrip = cms.bool(True),
+    associatePixel = cms.bool(True),
+    pixelSimLinkSrc = cms.InputTag("simSiPixelDigis"),
+    stripSimLinkSrc = cms.InputTag("simSiStripDigis"),
+)
+
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
-fastSim.toModify(quickTrackAssociatorByHits,
+fastSim.toReplaceWith(quickTrackAssociatorByHits, quickTrackAssociatorByHitsTrackerHitAssociator.clone(
     associateStrip = False,
     associatePixel = False,
-    useClusterTPAssociation = False
-)
-
-from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
-phase2_tracker.toModify( 
-    quickTrackAssociatorByHits,
-    pixelSimLinkSrc = cms.InputTag("simSiPixelDigis","Pixel"),
-    stripSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker")
```

Comment:
The starting point for these developments was this customization, which is
* incorrect: for phase 2, instead of `stripSimLinkSrc` one should set
```python
usePhase2Tracker = cms.bool(True),
phase2TrackerSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker")
```
* useless: These parameters are for `TrackerHitAssociator`, while by default `QuickTrackAssociatorByHits` uses `ClusterTPAssociation`

So this PR makes an attempt to remove the customization of useless and possibly confusing parameters.